internal/service/kafka/kafka_topic_create.go

```diff
@@ -9,19 +9,19 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
-// KafkaTopicCreateWaiter is used to create topics. Since topics are often
+// TopicCreateWaiter is used to create topics. Since topics are often
```

Comment:
I assume we are trying to remove the package name from the name of the struct, but it alters its meaning. The TopicWaiter can be a little misleading, and I suggest we keep Kafka in the name, but let's not export it any longer, and I think liners will be kind to it.

Reply:
I'm sorry, but it's forbidden to prefix methods in Go with it's package name, we will have to rename the package otherwise, see https://go.dev/blog/package-names:

> Avoid repetition. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.

Reply:
if we have this as an internal structure it is going to be OK